spec/account_client_spec.rb

```diff
@@ -67,9 +65,9 @@
 
     Algolia::AccountClient.copy_index!(@index_1, @index_3)
 
-    res = @index_3.search('');
-    res['nbHits'].should eq(1)
-    res['hits'][0]['objectID'].should eq('one')
+    res = @index_3.search('')
+    res['nbHits'].should eq(1500)
+    res['hits'][0]['objectID'].should eq('999')
```

Comment:
I think this test is confusing, since the index is not ordered; we don't know why '999' is first. Let's remove this line.

Reply:
What about adding a comment above the line then? At the end of the day I'd like to test that we retrieve the same objects and this is one way of doing it. 
Or I can save one last object with a clearer ID, change the `hitsPerPage` and retrieve this object instead

Reply:
if so, let's rather compare the whole list of 1500 objects then :) ensuring all numbers are retrieved.